cabot/metricsapp/models/elastic.py

```diff
@@ -1,6 +1,13 @@
 from django.db import models
-from cabot.metricsapp.api import create_es_client
-from .base import MetricsSourceBase
+from elasticsearch_dsl import Search
+import json
```

Comment:
nit: please club all the `import`s together and the `from ... import ...` together..